### PR TITLE
Renamed name of languages - phase 1

### PR DESCRIPTION
**Pull request by RyckRichards**
This PR solves #1670 

It renames

cmn -> Chinese (Mandarin) → Mandarin Chinese
nob -> Norwegian (Bokmål) → Norwegian Bokmål
nno -> Norwegian (Nynorsk) → Norwegian Nynorsk
afb -> Arabic (Gulf) → Gulf Arabic
aln -> Albanian (Gheg) → Gheg Albanian
bcl -> Bikol (Central) → Central Bikol
cmo -> Mnong, Central → Central Mnong
dtp -> Central Dusun → Kadazan Dusun
wuu -> Shanghainese → Wu Chinese
yue -> Cantonese → Yue Chinese

### CODE REVIEW

**Comment by trang** [comment]
@RyckRichards Can you please revert the changes `dtp`, `wuu` and `yue` in your pull request. As mentioned [in my comment](https://github.com/Tatoeba/tatoeba2/issues/1670#issuecomment-480632454), I do not feel very confident changing the name for these languages yet.

**Comment by RyckRichards** [comment]
Ok, sure :)

File: src/Lib/LanguagesLib.php [hunk 1]
```diff
@@ -260,13 +260,13 @@ public static function languagesInTatoeba()
...
                 // TODO to change when shanghainese will not be the only wu dialect
-                'wuu' => __d('languages', 'Wu Chinese'),
+                'wuu' => __d('languages', 'Shanghainese'),
                 'arz' => __d('languages', 'Egyptian Arabic'),
                 'bel' => __d('languages', 'Belarusian'),
                 'hun' => __d('languages', 'Hungarian'),
                 'isl' => __d('languages', 'Icelandic'),
                 'sqi' => __d('languages', 'Albanian'),
-                'yue' => __d('languages', 'Yue Chinese'),
+                'yue' => __d('languages', 'Shanghainese'),
```
Comment (trang) [inline src/Lib/LanguagesLib.php]: @RyckRichards This should be "Cantonese" instead of "Shanghainese".

Comment (RyckRichards) [reply src/Lib/LanguagesLib.php]: Oh sorry

**Pull request merged**